app/models/person.rb

```diff
@@ -113,4 +113,8 @@ def phone
   def notify_of_change?(person_responsible)
     EmailAddress.new(email).valid_address? && person_responsible.try(:email) != email
   end
+
+  def is? person
+    person && person.email == self.email
```

Comment:
I'm thinking if we shouldn't be using `===` here, because `email` has a `unique` constrain and there can't be 2 people with the same one anyway. Therefore maybe this could work?

```ruby
def is? person
  person === self
end
```